modules/project-docs/pages/sdk-release-notes.adoc

```diff
@@ -28,11 +28,68 @@ Java 11 has various enhancements like lambda local variable type inference, prof
     <dependency>
         <groupId>com.couchbase.client</groupId>
         <artifactId>java-client</artifactId>
-        <version>3.0.4</version>
+        <version>3.0.5</version>
     </dependency>
 </dependencies>
 ----
 
+== Version 3.0.5 (2 June 2020)
+
+Version 3.0.5 is the sixth release of the 3.0 series, bringing enhancements and bugfixes over the last stable release.
+
+http://packages.couchbase.com/clients/java/3.0.5/Couchbase-Java-Client-3.0.5.zip[Download] |
+http://docs.couchbase.com/sdk-api/couchbase-java-client-3.0.5/[API Reference] |
+http://docs.couchbase.com/sdk-api/couchbase-core-io-2.0.6/[Core API Reference]
+
+The supported and tested dependencies for this release are:
+
+* com.couchbase.client:**java-client:3.0.5**
+* com.couchbase.client:**core-io:2.0.6**
+* io.projectreactor:**reactor-core:3.3.5.RELEASE**
+* org.reactivestreams:**reactive-streams:1.0.3**
+
+=== Enhancements
+
+* http://issues.couchbase.com/browse/JVMCBC-852[JVMCBC-852]:
+Bumped Reactor to 3.3.5, Netty to 4.1.50.Final, and Jackson to 2.11.0
+* http://issues.couchbase.com/browse/JCBC-1637[JCBC-1637]:
+ConfigurationProvider now provides a seedNodes() Flux, allowing the application to be informed when all address resolution is complete and the final seed-nodes hostnames are known
+
+=== Bug Fixes
+
+* http://issues.couchbase.com/browse/JVMCBC-851[JVMCBC-851]:
+Reactive getAllReplicas methods will now honor a provided custom transcoder
+* http://issues.couchbase.com/browse/JCBC-1635[JCBC-1635]:
+Subscribing to DefaultConfigurationProvider.configs() will now correctly replay the current config
```

Comment:
This is not a bugfix, and also internal -- david needed it for some stuff

Reply:
The ticket is marked as a bug.  But sure, I'll remove this.